src/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ if (EXISTS "${CMAKE_SOURCE_DIR}/.git")
             "--git-dir=${CMAKE_SOURCE_DIR}/.git"
             describe
             --abbrev=4
+            --tags
```

Comment:
The last tag was not annotated, this fixes it
```diff
- v41-167-gfa5b
+ v42-136-gfa5b
```

Reply:
Ah, OK!

I guess it's soon time for a new release again.